packages/api/db/migration/20240207214919_create_animal_union_batch_internal_id_view_modify_animal_batch_tables.js

```diff
@@ -0,0 +1,83 @@
+/*
+ *  Copyright 2024 LiteFarm.org
+ *  This file is part of LiteFarm.
+ *
+ *  LiteFarm is free software: you can redistribute it and/or modify
+ *  it under the terms of the GNU General Public License as published by
+ *  the Free Software Foundation, either version 3 of the License, or
+ *  (at your option) any later version.
+ *
+ *  LiteFarm is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ *  GNU General Public License for more details, see <https://www.gnu.org/licenses/>.
+ */
+
+/**
+ * @param { import("knex").Knex } knex
+ * @returns { Promise<void> }
+ */
+export const up = async function (knex) {
+  try {
+    await knex.schema.alterTable('animal', (table) => {
+      table.string('photo_url');
+      table.dropChecks(['name_identifier_check']);
+    });
+
+    await knex.schema.alterTable('animal_batch', (table) => {
+      table.string('photo_url');
+      table.string('name').nullable().alter();
+    });
+
+    // Create function to assign internal_identifier to a record (NEW) to be inserted
```

Comment:
I think I still prefer to see this SQL code timestamped in an `sql/` directory (or `views/` directory) in case we do have to touch it in the future in another migration.

If we start doing that, I don't know -- I'm curious if you have a preference? -- if it would be better to save the raw SQL as JS string to make import trivial, or creating SQL files and using the full `fs.readFileSync(path.join(__dirname`... as in this [knex GitHub thread](https://github.com/knex/knex/issues?q=fs.readFileSync%28path.join%28__dirname%2C+in%3Acomments) but then having SQL in an .sql file, which gives syntax highlighting and maybe some other benefit I'm not considering.